account-management-service/src/main/java/org/bf2/srs/fleetmanager/spi/impl/AccountManagementServiceImpl.java

```diff
@@ -81,6 +83,8 @@ void init() {
 
     @Timed(value = Constants.AMS_DETERMINE_ALLOWED_INSTANCE_TIMER, description = Constants.AMS_TIMER_DESCRIPTION)
     @Audited
+    @Retry // 3 retries, 200ms jitter
+    @Timeout(3000) // 3000ms
```

Comment:
Maybe extract the value to a constant?

Reply:
Actually, we can even consider making it configurable

Reply:
Nothing against, I'll add it